src/3.x/configuration.md

```diff
@@ -46,23 +46,23 @@ Configuration | Explanation
 `design/search_engine_robots/default_robots` | Meta robots tag value
 `design/search_engine_robots/custom_instructions` | See [Robots.txt](configuration.html#robots-txt)
 
-If you need to access a Magento configuration you can use the [`@config` Blade Directive](theming.html#config) or the Rapidez facade `Rapidez::config()` which accepts the same parameters as the directive. For more advanced usage (like defining which scope) you can use:
+If you need to access a Magento configuration, you can use the [`@config` Blade Directive](theming.html#config) or the Rapidez facade `Rapidez::config()`, which accepts the same parameters as the directive. For more advanced usage (like defining which scope), you can use:
 ```php
 \Rapidez\Core\Models\Config::getValue($path, ConfigScopes::SCOPE_STORE, $scopeId, $options)
 ```
 
 ## Base URL
 
-Rapidez will be the frontend, Magento just as backend. So they'll need 2 different urls; we suggest to set the `web/secure/base_url` differently per scope:
+Rapidez will be the frontend, Magento just as the backend. So they'll need two different URLs; we suggest setting the `web/secure/base_url` differently per scope:
```

Comment:
```suggestion
Rapidez will become the frontend, Magento will only be used for the backend. Because of this, they'll need two different URLs. We suggest setting the `web/secure/base_url` differently per scope:
```